tests/data/codelist/tier_attribute/valid/variables.yaml

```
@@ -0,0 +1,11 @@
+- Primary Energy:
+    description: Total primary energy consumption
+    unit: EJ/yr
+- Primary Energy|{Fuel}|{Sector}:
+    description: Primary energy consumption of {Fuel}
+    unit: EJ/yr
+    tier: 1
```

Primary fuel consumption by definition does not have a sector.
```suggestion
- Primary Energy|{Fuel}:
    description: Primary energy consumption of {Fuel}
    unit: EJ/yr
    # this variable does not have a tier to test that no tier is added at initialization    
- Final Energy|{Fuel}:
    description: Final energy consumption of {Fuel}
    unit: EJ/yr
    tier: 1 
- Final Energy|{Fuel}|{Sector}:
    description: Final energy consumption of {Fuel} in {Sector}
    unit: EJ/yr
    tier: 1
```